weaviate/collections/classes/config_vectors.py

```diff
@@ -287,6 +289,43 @@ def multi2vec_jinaai(
             ),
         )
 
+    @staticmethod
+    def multi2vec_weaviate(
+        *,
+        image_field: str,
```

Comment:
Hey @dirkkul wdyt about this vs `image_fields: list[str]` for this context? 

Per @augustas1 we expect only 1 image prop to be vectorized for this use context (image-based document retrieval). But just wanted to check for consistency with other modules & preserving optionality if we add other models.

Reply:
> Hey @dirkkul wdyt about this vs image_fields: list[str] for this context?

fine with me, we can always replace it it `Union[str, list[str]]` later without breaking anything